### PR TITLE
Bids cpac dev

### PR DESCRIPTION
**Pull request by joshicola**
Added some package/module structure from work with Andy.

Tested in "dry-run" mode (without executing the c-pac command line).

### CODE REVIEW

**Comment by joshicola** [comment]
Default pipeline settings provide a 4 GB zipped payload.

This payload size is dependent on these settings. 

Working directory for c-pac is in /tmp/ and /scratch/.  These are not saved to the payload.

**Review by AndysWorth — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -5,19 +5,28 @@
 #
 
 # Start with the bids/cpac
-FROM fcpindi/c-pac:release-v1.4.3
+FROM fcpindi/c-pac:release-v1.4.1
 
 # Note the Maintainer
 MAINTAINER Michael Perry <michaelperry@flywheel.io>
```
Comment (lmperry) [inline Dockerfile]: Please modify to Flywheel <support@flywheel.io>

File: manifest.json [hunk 2]
```diff
@@ -7,14 +7,13 @@
     "url": "https://github.com/BIDS-Apps/CPAC",
     "source": "https://github.com/flywheel-apps/c-pac",
     "license": "Apache-2.0",
-    "flywheel": "0",
-    "version": "0.1.0-dev.13",
+    "version": "0.1.14_1.4.1",
     "custom": {
```
Comment (lmperry) [inline manifest.json:11]: Please add  the following to the ```custom``` key. 
```json
"flywheel": {"suite": "BIDS Apps"}
```

That will help keep these gears sorted in the UI.

File: run.py [hunk 3]
```diff
@@ -34,60 +27,55 @@
     try: # To download and validate BIDS 
         # for debugging:
         if context.destination['id'] == 'aex':
-            # give it the tome session
+            # give it an acquisition id, the below man not be valid
+            # for a given flywheel instance
             context.destination['id']='5d2761383289d60037e8b180'
```
Comment (lmperry) [inline run.py]: Can we remove these test blocks and instead use a custom config.json file?

**Review by lmperry — CHANGES_REQUESTED** [review]
Looks good. A couple of minor changes requested.

**Review by AndysWorth — APPROVED** [review]

**Pull request merged**